packages/lakka/libretro_cores/scummvm/package.mk

```diff
@@ -1,37 +1,41 @@
+
 PKG_NAME="scummvm"
-PKG_VERSION="2fb2e4c551c9c1510c56f6e890ee0300b7b3fca3"
-PKG_LICENSE="GPLv2"
+PKG_VERSION="a0554745e87361643f1ca3aa820a5073214de935"
+PKG_LICENSE="GPLv3"
 PKG_SITE="https://github.com/libretro/scummvm"
-PKG_URL="${PKG_SITE}.git"
+PKG_URL="$PKG_SITE.git"
 PKG_DEPENDS_TARGET="toolchain"
-PKG_LONGDESC="ScummVM with libretro backend."
+PKG_SHORTDESC="The ScummVM adventure game engine ported to libretro."
+PKG_LONGDESC="ScummVM is a program which allows you to run certain classic graphical point-and-click adventure games, provided you already have their data files."
 PKG_TOOLCHAIN="make"
-
-PKG_MAKE_OPTS_TARGET="-C ../backends/platform/libretro/build/"
+PKG_LR_UPDATE_TAG="yes"
```

Comment:
Do you tag "stable" releases in the repository? If so, then this is ok, otherwise we should update to latest commit - this line needs to be removed

Reply:
yes, I'm using [tags ](https://github.com/libretro/scummvm/tags) to mark "stable" points